src/makeCalculator.js

```diff
@@ -5,6 +5,39 @@
  */
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+
+    add(num) {
+      this.result += num;
+    },
```

Comment:
Methods must return `this` to allow chaining.
This will allow the methods to be called one after the other.
Fix it everywhere.

Example:
```suggestion
    add(num) {
      this.result += num;
      return this;
    },
```